packages/client/package.json

```diff
@@ -13,10 +13,12 @@
     "url": "https://github.com/ParabolInc/parabol/issues"
   },
   "scripts": {
+    "format": "yarn typecheck && yarn lint && yarn prettier",
     "storybook": "start-storybook -p 6006 -c .storybook",
     "build-storybook": "build-sFtorybook",
-    "lint": "eslint --fix ./**/*.{ts,tsx}",
+    "lint": "eslint --fix \"**/*.{ts,tsx}\"",
```

Comment:
fixed glob pattern. now we have like 600 errors that we'll want to fix up at a later time